pkg/workloads/console/controller.go

```diff
@@ -423,11 +424,30 @@ func (r *reconciler) buildJob(template *workloadsv1alpha1.ConsoleTemplate) *batc
 	// 1 pod per job.
 	backoffLimit := int32(0)
 
+	// jobName: Add console postfix and truncate name no longer than 49
+	// characters (63-14) (14 characters '-console-4mh4z')
+	jobName := fmt.Sprintf("%s-%s", stringTruncate(r.name.Name, 49), "console")
+
+	// Merged labels from the console template and console
+	// In case of conflits second label set wins
+	jobLabels := labels.Merge(csl.Labels, template.Labels)
+	jobLabels = labels.Merge(jobLabels,
+		map[string]string{
+			"console-name": stringTruncate(csl.Name, 63),
```

Comment:
Minor: This method should probably be called `truncateString`, for consistency with `sanitiseLabel`